Oneshot/lean4-in/lake-manifest.json

```diff
@@ -1,7 +1,9 @@
 {"version": 4,
- "packagesDir": "../../lake-packages",
+ "packagesDir": "lake-packages",
```

Comment:
was this supposed to be changed?

Reply:
I think `lake update` changed this.  AFAICT the value is only used for git packages, but everything uses relative paths here so it doesn't matter.